app/src/main/java/com/github/houseorganizer/houseorganizer/panels/InfoActivity.java

```diff
@@ -51,6 +61,19 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    public void changeNotesHousehold(View view){
+        EditText editNotes = findViewById(R.id.editTextHouseholdNotes);
```

Comment:
Why not use notesTextView? This will enhance the user experience for example in the case the user just wants to add something to the original notes.

Reply:
This would be impossible to implement without refactoring everything. Instead, When loading the notes I put the same text in the edit text box, which achieve the same "appending text" functionality

Reply:
I understand

Reply:
Took 1 hour to change the tests and everything but is done